dev-support/bump-accord.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+bin="$(cd "$(dirname "$0")" > /dev/null; pwd)"
+home="$(cd "$bin"/.. > /dev/null; pwd)"
+
+_main() {
+  git submodule status accord
+  echo "Is this the correct SHA? [y/n; default=y]"
+  read correct
+  if [[ "${correct:-y}" != "y" ]]; then
+    echo "Please update Accord's SHA and try again"
+    exit 1
+  fi
+  git commit -m "Change Accord to $(cd accord; git log -1  --format='%h: %B')" accord
```

Comment:
Worth mentioning the pre-commit hook will push the submodule in a comment here?

Reply:
I don't feel so, its also something you can opt-out of...